app/Http/Controllers/ClassesController.php

```diff
@@ -3,48 +3,65 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
-use App\Models\Class1;
- //use App\Http\controllers\time;
+use App\Models\class1;
+
+ 
 class ClassesController extends Controller
 {
+   
     /**
      * Display a listing of the resource.
      */
     public function index()
     {
-        //
+        $classes = class1::get();
+      return view('classes',compact('classes'));
     }
 
     /**
      * Show the form for creating a new resource.
      */
     public function create()
     {
-        return view('add_classes');
+         return view('add_classes');
     }
 
     /**
      * Store a newly created resource in storage.
      */
     public function store(Request $request)
     {
-         //dd($request);
-         $classname    ='swimming';
-         $capacity     = 40;
-         $price        = 400;
-         $isfilled     = true;
-         //format time
-         $timefrom = date('H:i:s', strtotime('10:02:03'));
-         $timeto = date('H:i:s', strtotime('12:50:05'));
-         Class1::create([
-             'classname'=> $classname,
-             'capacity' => $capacity,
-             'price'    => $price,
-             'isfilled' => $isfilled,
-             'timefrom' => $timefrom,
-            'timeto'    => $timeto,
+        //task5
+        $data =[
+            'classname'=>$request->classname,
+            'capacity'=>$request->capacity,
+            'price'=>$request->price,
+            'isfilled' =>isset($request->isfilled),
+            // 'isfilled' =>($request->isfilled === 'on' )? 1 : 0,
+            'timefrom' => date($request->timefrom, time()), 
```

Comment:
there is no need for formatting here ya esraa, there is a smarter solution use in frontend input of type time